SConstruct

```diff
@@ -110,6 +110,10 @@ opts.Add('p', "Platform (alias for 'platform')", '')
 opts.Add('platform', "Target platform (%s)" % ('|'.join(platform_list), ), '')
 opts.Add(EnumVariable('target', "Compilation target", 'debug', ('debug', 'release_debug', 'release')))
 opts.Add(EnumVariable('optimize', "Optimization type", 'speed', ('speed', 'size')))
+
+opts.Add(EnumVariable('renderer', "Renderer type", 'vulkan', ('vulkan', 'opengl')))
```

Comment:
AFAIK the plan is to provide both OpenGL ES 2.0 / GL 2.1 and Vulkan support in the same binary, so I don't think it should be a build option (or there should be a "all" option which would be the default value).

I see that Vulkan support raises min macOS version requirements though, so we should see whether we want to keep supporting pre 10.11 versons and have this build option, or drop support for older versions and have Vulkan + OpenGL for everyone.

Reply:
Using both will require some extra changes to all platforms.

Currently, OpenGLES 2 stuff is not functional, will it be reimplemented as `RenderingDevice` and use `RasterizerRD` or use existing rasterisers?

Reply:
I removed `renderer` option, and added runtime video driver checks assuming will use `RenderingDevice` (I have tested GL context part compiling, but disabled it for now as there's no `RenderingDevice` for GL). Any future changes (like auto fallback from Vulkan to GL) to the platform probably should wait for the multiple window implementation. Some of Vulkan / GL init could be done only after window creation and window creation have driver specific stuff.